CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+    - [Fork the Repository](#fork-the-repository)
+    - [Clone the Repository](#clone-the-repository)
+2. [Making Contributions](#making-contributions)
+    - [Branches](#branches)
+    - [Coding Standards](#coding-standards)
+    - [Commit Messages](#commit-messages)
+    - [Testing](#testing)
+3. [Submitting Changes](#submitting-changes)
+    - [Pull Requests](#pull-requests)
+    - [Issues](#issues)
+4. [Community and Communication](#community-and-communication)
+5. [Code of Conduct](#code-of-conduct)
+
+## Getting Started
+
+### Prerequisites
+
+Before you start contributing, please make sure you have the following:
+
+- [Git](https://git-scm.com/) installed
+- [A GitHub account](https://github.com/)
+- A text editor or integrated development environment (IDE) of your choice
+- Knowledge of the programming language(s) used in this project
+
+### Fork the Repository
+
+Fork our repository by clicking the "Fork" button on the top right of the repository page. This will create a copy of the project in your GitHub account.
+
+### Clone the Repository
+
+After forking the repository, clone it to your local machine using the following command, replacing `[your-username]` with your GitHub username:
+
+```bash
+git clone https://github.com/[your-username]/[repository-name].git
+```
+
+## Making Contributions
+
+### Branches
+
+Create a new branch for your work. It's important to keep your changes isolated from the `main` or `master` branch to prevent conflicts. Name your branch descriptively, such as `feature/your-feature-name` or `bugfix/issue-number`.
+
+```bash
+git checkout -b feature/your-feature-name
+```
+
+### Coding Standards
+
+Please follow the coding standards and style guide of the project. If there are no established standards, maintain consistency with the existing codebase.
+
+### Commit Messages
+
+Write clear, concise, and meaningful commit messages. A good commit message should:
+
+- Begin with a verb in the imperative mood (e.g., "Add," "Fix," "Update").
+- Be concise and focused on a single change.
+- Reference related issues or pull requests, if applicable.
+
+Example: "Fix #123: Add validation for user email input"
+
+### Testing
+
+Make sure your changes do not break existing functionality. Write tests for new features and update existing tests as needed.
+
+## Submitting Changes
+
+### Pull Requests
+
+When your contribution is ready, open a pull request (PR) from your branch to the main repository's branch (e.g., `main` or `master`). Be sure to:
+
+- Give the PR a descriptive title.
+- Provide a clear and detailed description of your changes.
+- Reference related issues, if any.
+- Be prepared to respond to feedback and make necessary adjustments.
+
+### Issues
+
+If you find a bug, have a suggestion, or want to work on an issue, please check the project's issue tracker. If the issue doesn't exist yet, feel free to open a new one to discuss the problem or feature request.
+
+## Community and Communication
```

Comment:
I don't have any discord, better write that reach out through my mail.

Reply:
remove this community and give my mail for communication (anand00rohan@gmail.com)